t/failed_request.t

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use Test::More;
+use Test::Exception;
+use lib 'lib';
+use_ok('Experian::IDAuth');
+
+# clean up
+system "rm -rf /tmp/proveid/";
```

Comment:
Nah... you don't check errors and tests might fail. It's better to have somehtin like

```perl
$tmp_dir = Path::Tiny->tempdir(CLEANUP => 1);
```

and assing the `$tmp_dir` to ` Experian::IDAuth`. It will be automattically cleaned whne `$tmp_dir goes out of scope. Also it will make possible to execut tests in parallel.